src/bot/cogs/sgc.py

```
@@ -0,0 +1,419 @@
+            webhook = discord.utils.get(webhooks, name="SharkBot-GlobalChat")
+            if not webhook:
+                webhook = await channel_obj.create_webhook(name="SharkBot-GlobalChat")
+
+            await self.bot.api.globalchat_update_connect_channel(
+                channel_id=str(channel_id),
+                guild_id=str(guild_id),
+                room_name=channel_info["room_name"],
+                webhook_url=webhook.url
+            )
+            return webhook.url
+        except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)